//
// This file is part of the WoW++ project.
// 
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 2 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
// GNU General Public License for more details.
// 
// You should have received a copy of the GNU General Public License
// along with this program; if not, write to the Free Software 
// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
//
// World of Warcraft, and all World of Warcraft or Warcraft art, images,
// and lore are copyrighted by Blizzard Entertainment, Inc.
// 

syntax = "proto2";
package wowpp.proto;

message AreaTriggerEntry {
	required uint32 id = 1;
	required string name = 2;
	required uint32 map = 3;
	required float x = 4;
	required float y = 5;
	required float z = 6;
	optional float radius = 7;
	optional float box_x = 8;
	optional float box_y = 9;
	optional float box_z = 10;
	optional float box_o = 11;
	required uint32 targetmap = 12;
	required float target_x = 13;
	required float target_y = 14;
	required float target_z = 15;
	required float target_o = 16;
	optional uint32 questid = 17;
	optional bool tavern = 18;
}

message AreaTriggers {
	repeated AreaTriggerEntry entry = 1;
}
